pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -148,13 +148,13 @@ func validateTopologyKey(p *field.Path, v string) field.ErrorList {
 }
 
 func validateWhenUnsatisfiable(p *field.Path, v v1.UnsatisfiableConstraintAction) *field.Error {
-	supportedScheduleActions := sets.NewString(string(v1.DoNotSchedule), string(v1.ScheduleAnyway))
+	supportedScheduleActions := sets.New[string](string(v1.DoNotSchedule), string(v1.ScheduleAnyway))
```

Comment:
This and all other `sets.New` invocations with parameters can be simplified:

```suggestion
	supportedScheduleActions := sets.New(string(v1.DoNotSchedule), string(v1.ScheduleAnyway))
```

Reply:
all done